<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Przelicz</class>
 <widget class="QWidget" name="Przelicz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kantor</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>266</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="2" column="0">
       <widget class="QPushButton" name="przycisk_dodaj">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>25</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #2a9df4;
color: white;
border: 1px solid #2a9df4;
border-radius: 10px;
}

QPushButton:hover{
background-color: #187bcd;
}</string>
        </property>
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="przycisk_wymien">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>21</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #2a9df4;
color: white;
border: 1px solid #2a9df4;
border-radius: 10px;
}

QPushButton:hover{
background-color: #187bcd;
}</string>
        </property>
        <property name="text">
         <string>Przelicz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>51</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0.5px solid #D3D3D3;
border-radius: 10px 10px 10px 0;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_2">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>31</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: 	#D3D3D3;
</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>WYBIERZ WALUTĘ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PLN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EUR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GBP</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>31</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: 	#D3D3D3;
</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>WYBIERZ WALUTĘ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PLN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EUR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CHF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GBP</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>201</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0.5px solid #D3D3D3;
border-radius: 10px 10px 10px 0;</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
